ADBView.sublime-settings

```diff
@@ -13,7 +13,7 @@
     "adb_snap_lines": 5,
 
     // Max number of lines in the adb logcat buffer
-    "adb_maxlines": 20000,
+    "adb_maxlines": 200000,
```

Comment:
200000 is too much. In my experience, ADBView can barely handle such a number of lines. So I suggest to change this value to 100000.